Source/WebInspectorUI/UserInterface/Views/DOMTreeElement.js

```diff
@@ -2017,30 +2018,40 @@ WI.DOMTreeElement = class DOMTreeElement extends WI.TreeElement
             this._layoutBadgeElement = null;
         }
 
-        if (!this.representedObject.layoutContextType)
-            return;
+        if (this._scrollBadgeElement) {
+            this._scrollBadgeElement.remove();
+            this._scrollBadgeElement = null;
+        }
 
-        this._layoutBadgeElement = this.title.appendChild(document.createElement("span"));
-        this._layoutBadgeElement.className = "layout-badge";
+        if (this.representedObject.layoutFlags.includes(WI.DOMNode.LayoutFlag.Scrollable)) {
+            this._scrollBadgeElement = this.title.appendChild(document.createElement("span"));
+            this._scrollBadgeElement.className = "layout-badge";
+            this._scrollBadgeElement.textContent = WI.unlocalizedString("scroll");
```

Comment:
I think we should use a `WI.UIString("scroll")` for this instead.  "grid" and "flex" are unlocalized because they directly correspond to CSS things (and are opt-in), whereas "scroll" is kinda more of a general idea